Cargo.toml

```diff
@@ -24,9 +24,12 @@ der = { version = "=0.6.0-pre.3", optional = true, default-features = false }
 generic-array = { version = "0.14", optional = true }
 rand_core = { version = "0.6", optional = true }
 rlp = { version = "0.5", optional = true, default-features = false }
+serde_ = { version = "1", package = "serde", optional = true, default-features = false }
+serdect = { version = "0.1", optional = true, default-features = false }
```

Comment:
`serdect` re-exports `serde` as `serdect::serde`.

Is there some other reason you're pulling it in as an explicit crate dependency?

Reply:
Yeah: I forgot :laughing:.
I feel very out-of-touch after such a long break.

Reply:
Done.